resist/shocal-dashboard.js

```diff
@@ -0,0 +1,51 @@
+var app = new Vue({
+  el: "#app",
+  data: {
+    latest: [],
+    currentPage: 1,
+    defaultSortDirection: "desc",
+    sortIcon: "arrow-up",
+    sortIconSize: "is-small",
+    columns: [
+      {
+        field: "orderID",
+        label: "Order No.",
+        numeric: true,
+      },
+      {
+        field: "customer.fullname",
+        label: "Customer",
+      },
+      {
+        field: "created",
+        label: "Date",
+        searchable: true,
+      },
+    ],
+  },
+  methods: {
+    fetchLatest() {
+      fetch(
+        "https://www.shocal.org/developers/apis/peepl/v1/data?apikey=Xau08wz65Sd12BaX5lU6B1W0lHyMD52167q5i"
```

Comment:
@zarino I'm not sure that this should be published in the public repo. Any ideas how we can store it more securely? And I will get Max to cycle the key.

Reply:
Oh no! I figured it was ok, as [you committed it here](https://github.com/itsaboutpeepl/itsaboutpeepl.com/commit/117067b2ded87187c390d81e2c87ddd00791fad2) on 2nd Feb.

I guess ideally we should store this in either a WordPress `option` (yuck) or in an environment variable on the server.

Might not be worth doing either, though, if you’d rather just get this all set up in a self-contained mini site anyway.